rust-bins/docs/user-cli.md

```diff
@@ -28,6 +28,17 @@ user_cli generate-request --cryptographic-parameters cryptographic-parameters.js
                           --id-use-data-out id-use-data.json \ # data that enables use of the identity object
                           --request-out request.json # request to send to the identity provider
 ```
+
+For Testnet woule be the following: 
```

Comment:
```suggestion
For Testnet would be the following: 
```

Reply:
This is also n the "Version 0 flow" section. If we are to add this we need to add one also in the V1 section.